### ISSUE TITLE
Error while running CaPTk BraTSPipeline on CBICA Cluster

### ISSUE THREAD

**Issue by sherjeel275** [CBICA/CaPTk#1400]
**Describe the bug**
I am trying to run BraTSPipeline from CBICA cluster. I get the following error:
![image](https://user-images.githubusercontent.com/52645980/109358070-28f8a500-7851-11eb-87c7-c6b64eb57504.png)

**To Reproduce**
1. First, went to my comp_space
2. Then ran the 'captk BraTSPipeline' command and provided as input 4 nifti files (t1, t1ce, t2, flair)
3. The processing finished early due to the error shown above

**Expected behavior**
I ran the BraTS Pipeline locally on my computer through the terminal. However, everything ran properly unlike on the cluster. I should have a directory called "dmOut_tumor" and brainTumorMask files.

**Screenshots**
![image](https://user-images.githubusercontent.com/52645980/109358478-d1a70480-7851-11eb-9796-ec0ca81b926a.png)

**Expected Output:**

![image](https://user-images.githubusercontent.com/52645980/109358806-58f47800-7852-11eb-8293-bf509b327715.png)

**CaPTk Version**
On the cluster I loaded the latest version of captk (version 1.8.1).

**Desktop (please complete the following information):**
 - I'm using a Macbook Pro.

**Comment by sarthakpati** [comment]
Hi, 

Could you paste the full command you used? Additionally, could you post the result of the command ```which BraTSPipeline```?

Cheers,
S

**Comment by sherjeel275** [comment]
Hi Sarthak,

Thanks for the quick response! Here is the command I used:

`captk BraTSPipeline -t1 $PWD/PT_1ZAWNGWT/PT_1ZAWNGWT_T1.nii.gz -t1c $PWD/PT_1ZAWNGWT/PT_1ZAWNGWT_T1CE.nii.gz -t2 $PWD/PT_1ZAWNGWT/PT_1ZAWNGWT_T2.nii.gz -fl $PWD/PT_1ZAWNGWT/PT_1ZAWNGWT_FLAIR.nii.gz -o $PWD/output`

Here's the result of `which BraTSPipeline`:
`/cbica/software/lab/captk/centos7/1.8.1/bin/BraTSPipeline`

Let me know if you need any additional information.

Cheers,
Sherjeel

**Comment by sarthakpati** [comment]
Hi Sherjeel,

The version and command are all right. Would it be possible for you to send me an example of a subject that failed? 

Cheers,
Sarthak

**Comment by sherjeel275** [comment]
Hi Sarthak,

By 'example of subject', do you mean the 4 nifti files that I used?

Best,
Sherjeel

**Comment by sarthakpati** [comment]
Yup, that is correct. You can email it to me, if it is more appropriate: patis [at] upenn.edu

**Comment by sherjeel275** [comment]
Just sent the files. Let me know if you were able to receive them?

**Comment by sarthakpati** [comment]
I'll keep you posted.

**Comment by sherjeel275** [comment]
Hi Sarthak,

Have you been able to look into the bug?

Thanks,
Sherjeel

**Comment by sarthakpati** [comment]
Hi Sherjeel,

I have not received anything. Could you send it to me again? 

Cheers,
Sarthak

**Comment by sarthakpati** [comment]
Hi Sherjeel,

I am able to run this without an issue on my local (Windows) machine. My VPN has been very spotty today so I cannot run on the cluster to check.

Would it be possible for you to run this on Windows? Even if you have a network-mounted location on your system, it should work.

Cheers,
Sarthak

**Comment by sherjeel275** [comment]
Hi Sarthak,

I have a Macbook and I was able to run the command locally without any issue. The problem is with running it on the cluster. I need to get it to run on the cluster because in the future I will be running this command on a large dataset and for that I will need to submit jobs to the cluster. Let me know if you are able to run the command on the cluster tomorrow.

Thanks,
Sherjeel

**Comment by sarthakpati** [comment]
Okay, I'll keep you posted.
